src/util/versiondownloader.cpp

```
@@ -17,7 +20,17 @@ VersionDownloader::~VersionDownloader()
 
 void VersionDownloader::performCheck()
 {
-    auto versionManifest = QUrl("https://api.github.com/repos/ModbusScope/ModbusScope/releases/latest");
+    QString versionString = Util::currentVersion();
+#ifdef DEBUG
+    versionString += QString("-%1-%2").arg(GIT_BRANCH, GIT_COMMIT_HASH);
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)